documentation/BlockConstruct.md

```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright (c) 2019, Arm Ltd.  All rights reserved.
+-->
+# Design proposal for Block Construct statement semantic analysis
+# The implementation of Block construct is based on the Section 11.1.4 of
+# Fortran 2018 standards Document
```

Comment:
The above two lines probably shouldn't be part of the header, only the line starting "Design proposal...".